### ISSUE TITLE
UI Crash on hovering over tag type annotation conflict with GT in Review section

### ISSUE THREAD

**Issue by Viditagarwal7479** [cvat-ai/cvat#7749]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Steps to Reproduce

1. Create a task
2. Annotate having annotation of type `tag`
3. Create a GT task 
4. Go into the `review` section, enable GT conflict
5. Hover over a GT conflict corresponding to the annotation of type `tag`
6. The UI crashes with an error

### Expected Behavior

The label with conflict should be highlighted.

### Environment

```Markdown
`git log -1`

commit 67845872c67849f975a25ce9911a7ef6348931f5 (HEAD -> quality_report, origin/quality_report)
Merge: 72fe5af39 55ea04b22
Author: Maxim Zhiltsov <zhiltsov.max35@gmail.com>
Date:   Wed Apr 10 20:42:37 2024 +0300

    Merge branch 'develop' into quality_report
```

**Comment by klakhov** [comment]
No, I dont think so. GT tags display was introduced in #7774. Currently there is no crush, conflicts are highlighted as expected.
